backend/controller/AccountController.ts

```diff
@@ -318,7 +327,7 @@ export class AccountController {
         if (skipFurtherChecks) {
             return true;
         }
-        if (name.toLowerCase() == this.defaultUsername) {
+        if (name.toLowerCase() == this.defaultAdminName) {
```

Comment:
Do we want to prevent `admin` + any suffix? Since we might generate admins with suffixes